### PR TITLE
autopep8 whitespace

### PR DESCRIPTION
**Pull request by thorade**
This is the first batch of autopep8 changes for #1612, the exact command used was
```
autopep8 --in-place --recursive --max-line-length=200 --exclude="externals" --select="E101,W1,W2" .
```

Note: I changed the commits (and force-pushed to the branch) so that now there are only those whitespace-changes that git can hide. To verify, append `?w=1` to the URL of the diff page:
https://github.com/CoolProp/CoolProp/pull/1613/files
https://github.com/CoolProp/CoolProp/pull/1613/files?w=1

### CODE REVIEW

File: Web/develop/web2py_online.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 ---------------------------
```
Comment (thorade) [inline Web/develop/web2py_online.py]: Should this be a .py file?

File: dev/Tickets/1443.py [hunk 2]
```diff
@@ -108,19 +106,19 @@ def calc_isolines(fluid="HEOS::water", kind_a="P", range_a=[1e6,10e6], kind_b="T
         elif CUR_CONF==CONF_PYT_DIR:
             start = timer()
             for i,two in np.ndenumerate(vector_in1):
-                try: 
+                try:
                     state.update(input_pair, two, one)
                     for kind_o_single in kind_o_index:
                         tmp_result = state.keyed_output(kind_o_single)
                 except:
                     ++err_count
```
Comment (thorade) [inline dev/Tickets/1443.py]: What is this? It is strangely highlighted.

**Comment by thorade** [comment]
@jowr should i change the extension of Web/develop/web2py_online.py to .rst?

**Comment by jowr** [comment]
I have no idea about web2py - but yes, if it is an rst file, why no call it that?

**Comment by ibell** [comment]
Yeah, that should be a RST file

**Pull request merged**